src/mumble/UserLocalVolumeDialog.h

```diff
@@ -45,16 +47,21 @@ class UserLocalVolumeDialog : public QDialog, private Ui::UserLocalVolumeDialog
 
 		/// The session ID for the user that the dialog is changing the volume for.
 		unsigned int m_clientSession;
+
 		/// The user's original adjustment (in dB) when entering the dialog.
 		int m_originalVolumeAdjustmentDecibel;
+		QMap<unsigned int, UserLocalVolumeDialog*> *m_qmuservolTracker;
 
 	public slots:
+		void closeEvent(QCloseEvent *event);
 		void on_qsUserLocalVolume_valueChanged(int value);
 		void on_qsbUserLocalVolume_valueChanged(int value);
 		void on_qbbUserLocalVolume_clicked(QAbstractButton *b);
+		void reject();
 
 	public:
-		UserLocalVolumeDialog(unsigned int sessionId = 0);
+		UserLocalVolumeDialog(unsigned int sessionId = 0,
+                                      QMap<unsigned int, UserLocalVolumeDialog*> *qmuservolTracker = NULL);
```

Comment:
The class can't work with qmuservolTracker == NULL, so don't use a default value here.